hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/multimap/MultiMapClearMessageTask.java

```diff
@@ -52,12 +52,12 @@ protected Object reduce(Map<Integer, Object> map) {
         for (Object affectedEntries : map.values()) {
             totalAffectedEntries += (Integer) affectedEntries;
         }
+        updateStats(LocalMapStatsImpl::incrementOtherOperations);
```

Comment:
I normally would expect stats update on the operation itself, e.g. `ClearOperation` for this. For example see MapPutMessageTask. Hence, I don't understand why you changed it to do in the message task. By putting it in the operation, it also guarantees that both server and client side APIs work the same way.

Reply:
These operations are called by creating a new operation instance for each partition. Because I don't want this operations to update the map stats multiple times, I added this into reduce() section. See the similar map operation: [MapPutAllMessageTask](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/MapPutAllMessageTask.java#L79-L83)